docs/build_and_load.md

```diff
@@ -975,7 +975,9 @@ phg create-maf-vcf \
     --bed output/ref_ranges.bed \
     --reference-file output/updated_assemblies/Ref.fa \
     --maf-dir output/alignment_files \
-    -o output/vcf_files
+    -o output/vcf_files \
+    --metrics-file output/vcf_files/VCFMetrics.tsv \
```

Comment:
Seems like the absence of a metrics file would imply --skip-metrics

Reply:
@tcasstevens I think the metrics-file parameter is used to change the name and location of where this file will be printed.  Its absence means to us default location, but it will still be created unless skip-metrics is true.

Reply:
@tcasstevens @lynnjo is correct. It's set up this way because we wanted the default behavior to produce a metrics file, to encourage users to do QC as they go.